### PR TITLE
Update Wildlife Park 2 (fix preInstall)

### LINKED ISSUE #521: fix Wildlife Park
@Zemogiter 

in `Applications/Games/Wildlife Park 2/Local/script.js`

see https://app.codacy.com/app/PhoenicisOrg/Scripts/file/15838939989/issues/source?bid=6547561&fileBranchId=6547561#l11

It must be:
```Javascript
.preInstall(function(wine, wizard) {
```
instead of:
```Javascript
.preInstall(function(wine, wizzard) {
```

### PR DESCRIPTION
**Pull request by Zemogiter**
fixes #521

### CODE REVIEW

**Comment by plata** [comment]
What's the purpose of this pull request?

**Comment by Zemogiter** [comment]
#521 you told me to fix the `.preInstall` line.

**Comment by plata** [comment]
Please follow these steps:
1. merge the up-to-date master
2. check in Codacy if the pull request fixes the issue
3. use a meaningful title for the pull request (follow guidelines in README)
4. add "fixes #521" to the description to close the issue once the pull request is merged

**Comment by Zemogiter** [comment]
Ready for review.

**Comment by plata** [comment]
This doesn't fix the Codacy issue. See #521.

The pr title should be something like "Update Wildlife Park 2 (fix preInstall)" (see README).

File: Applications/Games/Wildlife Park 2/Steam/script.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ new SteamScript()
     .applicationHomepage("www.wildlifepark2.com")
     .author("Zemogiter")
     .appId(264710)
-    .preInstall(function(wine, wizard) {
+    .preInstall(function(wine,wizard) {
```
Comment (plata) [inline Applications/Games/Wildlife Park 2/Steam/script.js]: Revert this (there should be a space after `,`).

**Comment by Zemogiter** [comment]
Ready for review.

**Comment by plata** [comment]
There are no changes.

**Pull request merged**